pages/index.js

```diff
@@ -1,14 +1,13 @@
+import { useCallback } from "react";
 import ArticleListContainer from "../components/ArticleListContainer";
-import { getAllArticles } from "../services/articleService";
+import { useState, useEffect } from "react";
 
-export async function getServerSideProps() {
-  const articles = await getAllArticles();
+export default function Home() {
+  const [articles, setArticles] = useState([]);
 
-  return {
-    props: { articles: articles },
-  };
-}
-export default function Home({ articles }) {
+  useEffect(() => {
+    setArticles(JSON.parse(localStorage.getItem("articles")) ?? []);
```

Comment:
You could use the localStorage hook that Thomas posted a few days ago. No need of useEffect on your index page then :)

Reply:
yeah i shall use it u re right. Thank you!